zipkin-cassandra/src/test/scala/com/twitter/zipkin/storage/cassandra/CassandraSpanStoreSpec.scala

```diff
@@ -36,4 +40,23 @@ class CassandraSpanStoreSpec extends SpanStoreSpec {
   override lazy val store = new CassandraSpanStore(new Repository(keyspace, cluster))
 
   override def clear = cluster.connect().execute("DROP KEYSPACE IF EXISTS " + keyspace)
+
+  override def setTimeToLive() {
+    ready(store(Seq(span1)))
+    ready(store.setTimeToLive(span1.traceId, 1234.seconds))
+
+    for( i <- 1 to 100) {
+      // @xxx Repository.storeXXX() methods write asynchronously but don't return a Future so we can't reliably test.
```

Comment:
Easiest way our would be to add doc to base span store, say it is
eventually consistent, and change the base test. I think many ops will end
up eventually consistent anyways. Regardless, I'm fine with this.